src/hooks/useHideBullet.ts

```diff
@@ -42,9 +44,14 @@ const useHideBullet = ({
       thought.value !== '=grandchildren' && attributeEquals(state, head(simplePath), '=bullet', 'None')
 
     /** Returns true if the bullet should be hidden because it is in table column 1 and is not the cursor. */
-    const hideBulletTable = () =>
-      !equalPath(simplePath, state.cursor) &&
-      attributeEquals(state, head(rootedParentOf(state, simplePath)), '=view', 'Table')
+    const hideBulletTable = () => {
+      // Don't hide bullets in context view
+      if (isInContextView) return false
```

Comment:
It would be a bit cleaner to place this condition in the existing return statement.

I try to avoid multiple consecutive return statements.